smtp-mail.cpp

```diff
@@ -54,6 +54,7 @@ void compose_payload(vector<std::string>* &payload, const EmailCfg *emailCfg, co
 {
 	payload->push_back("Date: " + string(getCurrTime()) + "\r\n");
 	payload->push_back("To: " + emailCfg->email_to_name + " <" + emailCfg->email_to + "> \r\n");
+	payload->push_back("Cc: " + emailCfg->email_cc_name + " <" + emailCfg->email_cc + "> \r\n");
```

Comment:
"Bcc" part also needs to be added here to payload?

Reply:
Bcc field is not to be seen by everyone except the sender. So if we add Bcc in the payload then it will be visible to all the recipients. That's why it's not included in the payload

Reply:
Ok, sounds good.

Reply:
Please leave an inline comment in the code, for future code readers/devs.